src/main/java/Duke.java

```diff
@@ -1,10 +1,48 @@
+import command.Command;
+import data.*;
```

Comment:
Minor nit-pick: would it be better to separate out the imports so people can know exactly what is imported?